ariadne/cardano/src/Ariadne/Wallet/UiAdapter.hs

```diff
@@ -97,7 +98,7 @@ toUiWalletDatas db = toUiWalletData <$> walletList
     toUiWalletData :: HdRoot -> UiWalletData
     toUiWalletData HdRoot{..} = UiWalletData
       { _uwdName = unWalletName _hdRootName
-      , _uwdId = _hdRootId
+      , _uwdId = let (AbstractHash x) = _fromDb $ unHdRootId $ _hdRootId in show x
```

Comment:
I think that using `sformat hashHexF` would be better, because it makes it clearer that hex format will be used (and because that's how `componentTokenRender` is defined for `TokenAddressHash`).

Reply:
It's even weirder now.
Better define one function in `ariadne-cardano`, which both `Glue` and `Knit` will use.